src/stylesheets/core/_utilities.scss

```diff
@@ -26,22 +26,12 @@
 // Unstyled list helper
 
 @mixin unstyled-list() {
-  display: block;
+  @include margin(0 null);
```

Comment:
So this rule will be expanded to:

```
margin-bottom: 0;
margin-top: 0;
```

I definitely don't think it's a good idea to mess with the webkit specific margin before rules. Those should be overridden with regular margin rules.